src/python/WMCore/JobStateMachine/ChangeState.py

```diff
@@ -581,7 +521,16 @@ def loadExtraJobInformation(self, jobs):
                 jobs[idx]["taskType"] = jobTask["type"]
                 jobs[idx]["jobType"] = jobTask["subtype"]
 
-    def completeCreatedJobsInformation(self, jobs, incrementRetry=False):
+    def completeCreatedJobsInformation(self, jobs, newstate, oldstate):
+        # If the new state is created it possible came from 3 locations:
+        # JobCreator in that case it comes with all the needed info
+        # ErrorHandler comes with the standard information of a WMBSJob
+        # RetryManager comes with the standard information of a WMBSJob
+        # Unpause script comes with the standard information of a WMBSJob
+        # For those last 3 cases we need to fill the gaps
```

Comment:
What do you mean by :
> fill the gaps

Reply:
information not yet loaded in the job object (see lines below), like job mask, retry counter, etc

Reply:
ok got it